app/assets/stylesheets/components/accessible_autocomplete_tweaks.scss

```diff
@@ -11,3 +11,18 @@
     }
   }
 }
+
+.flex {
```

Comment:
Can they be named more appropriately to indicate what they're for and reduce the chances of naming clashes? e.g. `.autocomplete-tweak__suggestion`, `.autocomplete-tweak__suggestion__main-section`, `.autocomplete-tweak__suggestion__closed-status`.

Reply:
Actually, they should probably be something other than `autocomplete-tweak__*` (`school-search__*`?) since they're used by the school search template itself. Probably we should also pull them into a different file, since they're not strictly about autocompletion.